mslib/mscolab/file_manager.py

```diff
@@ -250,6 +254,42 @@ def modify_user(self, user, attribute=None, value=None, action=None):
             db.session.commit()
         return True
 
+    def save_user_profile_image(self, user_id, image_file):
+        """
+        Save the user's profile image path to the database.
+        """
+        upload_folder = mscolab_settings.UPLOAD_FOLDER
+        file_token = secrets.token_urlsafe(20)
+
+        with fs.open_fs('/') as home_fs:
+            file_dir = fs.path.join(upload_folder, 'profile')
+            # Checking if directory exists. Create the directory if it does not exist
+            if '\\' not in file_dir:
```

Comment:
I don't quite understand the necessity for this distinction. "\\" is a perfectly valid character in a path on Linux and shouldn't silently be replaced with something else. If this is something for windows compatibility it should have a conditional based on if it is running on windows.

Reply:
windows

Reply:
So then this condition should be either `platform.system() == "Windows"` or `os.sep == "\\"`. Or we could define that the upload folder should always be specified with forward slashes, in which case I don't think any backslash could ever reach this point.